packages/components/src/components/PipelineRun/Run.scss

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Created https://github.com/tektoncd/dashboard/issues/451 to track removing the old file once the other containers are updated